src/data-publication/reports/tables/Aggregate1.jsx

```diff
@@ -59,99 +105,172 @@ const Aggregate1 = React.forwardRef((props, ref) => {
     return 0
   })
 
-  let colWidth = '5.7%'
+  return fixUnaggregatedDispositions(sorted)
+}
+
+export const buildCSVRowsAggregate1 = (report) => {
```

Comment:
I believe it would make sense to move the `buildCSVRowsAggregate1` function to a utility file since it isn't being used in this file.

What do you think?

Reply:
Done.